client/my-sites/plugins/plugin-details.jsx

```diff
@@ -89,7 +89,6 @@ function PluginDetails( props ) {
 		isRequestingForSites( state, siteIds )
 	);
 	const analyticsPath = selectedSite ? '/plugins/:plugin/:site' : '/plugins/:plugin';
-	const { localizePath } = useLocalizedPlugins();
```

Comment:
Removing this is unrelated to this PR, but I wanted to make the linter happy.

Reply:
This is used now https://github.com/Automattic/wp-calypso/pull/67140

Reply:
Doh! Added back.